src/main/java/com/mparticle/kits/RadarKit.java

```diff
@@ -39,9 +39,16 @@ private void tryTrackOnce() {
         String publishableKey = settings.get(KEY_PUBLISHABLE_KEY);
         mRunAutomatically = settings.containsKey(KEY_RUN_AUTOMATICALLY) && Boolean.parseBoolean(settings.get(KEY_RUN_AUTOMATICALLY));
 
-        Radar.initialize(context, publishableKey);
+        Radar.initialize(publishableKey);
+
+        Map<MParticle.IdentityType, String> identities = getUserIdentities();
+        String customerId = identities.get(MParticle.IdentityType.CustomerId);
+        if (customerId != null) {
+            Radar.setUserId(customerId);
```

Comment:
I noticed that down below in `setUserIdentity()` we don't check for null, but here we do, just wanted to make sure that's ok.

Reply:
`null` is definitely an acceptable value, it just clears the set userId. But I'm not sure what behavior we'd want here cc @nickpatrick thoughts?